recipe/build.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+set -ex
+
+# necessary for cross-compilation to point to the right env
+export PKG_CONFIG_PATH=$PREFIX/lib/pkgconfig
+
+# HACK: extend $CONDA_PREFIX/meson_cross_file that's created in
+# https://github.com/conda-forge/ctng-compiler-activation-feedstock/blob/main/recipe/activate-gcc.sh
+# https://github.com/conda-forge/clang-compiler-activation-feedstock/blob/main/recipe/activate-clang.sh
+# to use host python; requires that [binaries] section is last in meson_cross_file
+echo "python = '${PREFIX}/bin/python'" >> ${CONDA_PREFIX}/meson_cross_file.txt
+
+# meson-python already sets up a -Dbuildtype=release argument to meson, so
+# we need to strip --buildtype out of MESON_ARGS or fail due to redundancy
+MESON_ARGS_REDUCED="$(echo $MESON_ARGS | sed 's/--buildtype release //g')"
```

Comment:
Where does the original `MESON_ARGS` come from? conda-build? or some default conda-forge specific config environment? Shall we report this problem upstream?

Reply:
`MESON_ARGS` comes from conda-forge: https://conda-forge.org/docs/maintainer/knowledge_base/#how-to-enable-cross-compilation